index.html

```diff
@@ -218,6 +218,19 @@ <h3 class="sub-title">Skills <img class="arrow-icon" src="./resources/icon/side
                 </ul>
             </ul>
         </section>
+        
+        <section class="contact--section">
+          <div>
+            <form action="https://formspree.io/f/xwkjylde" method="post" class="form">
+                <h2 class="contact-me">Contact me</h2>
+                <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it!</p>
+              <input type="text" name="username" class="user" placeholder="Write your name here">
+              <input type="email" name="useremail" class="user" placeholder="Email address, phone number or Skype">
+              <textarea name="text" class="textarea" maxlength="500" minlength="1" cols="30" rows="10" placeholder="Write your message here ..."> </textarea>
```

Comment:
- [ ] Kindly make sure all input fields have the `required` attribute to activate the validation on submission. 
- [ ] Also please make sure the name input has a `maxlength="30"` attribute as this is part of the project requirements.